src/core/ComplexFilter/components/Chips/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
```

Comment:
Chips subcomponent handles generating the collection of Chips for the complex filter's selected options